### ISSUE TITLE
Web accessibility reader error

### ISSUE THREAD

**Issue by maxl13** [swimlane/ngx-datatable#498]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**

Pager component failes Chrome WAVE (Web accessibility evaluation).

**Expected behavior**

The Pager ellements would pass the readability of reader devices for the disabled. 

**Reproduction of the problem**

**What is the motivation / use case for changing the behavior?**

Make it more accessible to diabled users

**Please tell us about your environment:**

* **Table version:** 0.7.x

* **Angular version:** 2.0.x

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
All

By adding an "Aria-label" to the <a> tag of the pager.component.ts would fix the reader issues.
Example:

        <a
          href="javascript:void(0)"
          aria-label=”First Page”
          (click)="selectPage(1)">
          <i class="{{pagerPreviousIcon}}"></i>
        </a>

        <a
          href="javascript:void(0)"
          aria-label=”previous Page”
          (click)="prevPage()">
          <i class="{{pagerLeftArrowIcon}}"></i>
        </a>

        <a
          href="javascript:void(0)"
          aria-label=”Select Page”
          (click)="selectPage(pg.number)">
          {{pg.text}}
        </a>

        <a
          href="javascript:void(0)"
          aria-label=”Next Page”
          (click)="nextPage()">
          <i class="{{pagerRightArrowIcon}}"></i>
        </a>

        <a
          href="javascript:void(0)"
          aria-label=”Last Page”
          (click)="selectPage(totalPages)">
          <i class="{{pagerNextIcon}}"></i>
        </a>

**Comment by amcdnl** [comment]
dupe of https://github.com/swimlane/ngx-datatable/issues/257
